proof/refine/ARM_HYP/Ipc_R.thy

```diff
@@ -1563,8 +1563,8 @@ lemma makeArchFaultMessage_corres:
     apply (rule corres_split_eqr[OF _ asUser_getRestartPC_corres])
       apply (rule corres_split_eqr[OF _ corres_machine_op])
          apply (rule corres_trivial, simp)
-        apply (rule corres_underlying_trivial[OF addressTranslateS1CPR_no_fail])
-       apply (wp+, auto)
+        apply (rule corres_underlying_trivial)
+        apply (wp+, auto)
```

Comment:
These changes to ARM_HYP Refine appear in a commit tagged "lib proof:", which is not a tag I remember seeing before

Reply:
Looking over the history, I see that `lib proof:` has existed before, but it's confusing, if we're going that way I'd vote for `lib+proof:`

Reply:
In my mind I've changed lib to make `no_fail` use prefix-naming, and then updated all of the different proofs for anything that broke. While it turns out that this was the only change needed, I wouldn't want to label it specifically `ARM_HYP refine` since it conceptually isn't specific to just that.

Reply:
Oh, I'd also be happy with lib+proof.